### ISSUE TITLE
Preventing automatic write to firestore

### ISSUE THREAD

**Issue by kartik2206** [AnthonyNahas/ngx-auth-firebaseui#149]
I have my own cloud function set up for new account creation and I have some other fields to add to user doc as well so I would like to stop this automatic write to firestore. How can I do that ?

**Comment by AnthonyNahas** [comment]
this feature is in the pipeline and will be supported asap!

**Comment by AnthonyNahas** [comment]
I've just published ngx-auth-firebaseui v2.2.0 that supports now the ability to enable/disable autosync users' feature with firestore. All you have to do is to add an `NgxAuthFirebaseUIConfig` as 3rd params (via the `enableFirestoreSync` key) to the `NgxAuthFirebaseUIModule` module like below .. e.g: 

```typescript
 NgxAuthFirebaseUIModule.forRoot({
                    apiKey: 'your-firebase-apiKey',
                    authDomain: 'your-firebase-authDomain',
                    databaseURL: 'your-firebase-databaseURL',
                    projectId: 'your-firebase-projectId',
                    storageBucket: 'your-firebase-storageBucket',
                    messagingSenderId: 'your-firebase-messagingSenderId'
                }, 'your_firebase_project_name',
               {
               enableFirestoreSync: false,
               toastMessageOnAuthSuccess: true,
               toastMessageOnAuthError: true
              }),
```

thanks for contributing 👍

**Comment by kartik2206** [comment]
Thank you so much for this feature. Thank you 👍 :)
